src/components/table/table.module.scss

```diff
@@ -0,0 +1,113 @@
+@import 'src/assets/styles/variables/typography';
+
+$HEIGHT-SMALL: 44px;
+$HEIGHT-DEFAULT: 64px;
+$HEIGHT-LARGE: 80px;
+
+* {
+  box-sizing: border-box;
+}
+
+.table {
+  width: 100%;
+  max-width: 1200px;
+}
+
+.table-header {
+  display: flex;
+  width: 100%;
+  height: 32px;
+  align-items: center;
+}
+
+.table-row {
+  display: flex;
+  width: 100%;
+  height: $HEIGHT-DEFAULT;
+
+   .table-row-content {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    flex: 1;
+    box-shadow: var(--rp-ui-base-shadow-hover);
+    background-color: var(--rp-ui-base-bg-000);
```

Comment:
Why are base colors used on this file?

Reply:
As we don't support theming for this component right now, it is ok to have base colors.
I suppose we will use design tokens instead once they will be ready on UX side.